docs/kb/api-version-4/prospects.md

```diff
@@ -269,11 +267,11 @@ JSON Example:
             "first_name": "New first name",
             "last_name": "New last name"
         },
-        "some@email.com": {
+        "1337": {
```

Comment:
but, yes, I'll be adding a section that shows create being different from update/upsert.